### PR TITLE
fix: MD010/no-hard-tabs

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@nschonni : Thanks for your contribution! The author, @MikeJo5000, has been notified to review your proposed change.

File: docs/ide/customize-build-and-debug-tasks-in-visual-studio.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ If your codebase uses custom build tools that Visual Studio doesn't recognize, t
 
 Consider a codebase that consists of a single C# file called *hello.cs*. The *makefile* for such a codebase might look like this:
 
+<!-- markdownlint-disable MD010 -->
```
Comment (nschonni) [inline docs/ide/customize-build-and-debug-tasks-in-visual-studio.md]: Makefile is supposed to use tabs, so I added a specific exception

**Comment by ktoliver** [comment]
@gewarren Could you review the proposed changes? Thanks.

File: docs/azure/vs-azure-tools-cloud-service-project-managing-roles.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The following steps guide you through adding a web or worker role to an Azure cl
 
 1. Right-click the **Roles** node to display the context menu. From the context menu, select **Add**, then select an existing web role or worker role from the current solution, or create a web or worker role project. You can also select an appropriate project, such as an ASP.NET web application project, and associate it with a role project.
 
-	![Menu options to add a role to an Azure cloud service project](./media/vs-azure-tools-cloud-service-project-managing-roles/add-role.png)
+  ![Menu options to add a role to an Azure cloud service project](./media/vs-azure-tools-cloud-service-project-managing-roles/add-role.png)
```
Comment (gewarren) [inline docs/azure/vs-azure-tools-cloud-service-project-managing-roles.md]:
```suggestion
   ![Menu options to add a role to an Azure cloud service project](./media/vs-azure-tools-cloud-service-project-managing-roles/add-role.png)
```

This needs to be indented three spaces to appear as indented under the numbered list.

File: docs/azure/vs-azure-tools-cloud-service-project-managing-roles.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ The following steps guide you through removing a web or worker role from an Azur
 
 1. Right-click the node you want to remove, and, from the context menu, select **Remove**.
 
-	![Menu options to add a role to an Azure cloud service](./media/vs-azure-tools-cloud-service-project-managing-roles/remove-role.png)
+  ![Menu options to add a role to an Azure cloud service](./media/vs-azure-tools-cloud-service-project-managing-roles/remove-role.png)
```
Comment (gewarren) [inline docs/azure/vs-azure-tools-cloud-service-project-managing-roles.md]:
```suggestion
   ![Menu options to add a role to an Azure cloud service](./media/vs-azure-tools-cloud-service-project-managing-roles/remove-role.png)
```

**Comment by nschonni** [comment]
Looks like `docs/ide/how-to-report-a-problem-with-visual-studio-2017.md` got renamed. I'll try and rebase this later, unless you pull it internally before I do

**Comment by nschonni** [comment]
@gewarren i've rebased this for the conflict

**Comment by ktoliver** [comment]
@gewarren Thanks for reviewing. I'm going to pull the commit to the private repo to verify the formatting changes on staging before we merged.

**Comment by Jak-MS** [comment]
fixed merge conflicts and moved the commits in this PR to the private repo for validation and staging. closing this PR. thanks!

**Pull request closed without merging**